File: resource/java/com/template/BuildCondition.java
package com.template;
import java.io.DataInputStream;
public class BuildCondition implements TemplateInterface
{
	public String c_id;
	public String build_name;
	public short build_type;
	public short build_num;
	public short condition_level;
	public void read(DataInputStream in)throws Exception
	{
		c_id=in.readUTF();
		build_name=in.readUTF();
		build_type=in.readShort();
		build_num=in.readShort();
		condition_level=in.readShort();
	}
}

File: resource/cpp/com/template/BuildCondition.cpp
#include "BuildCondition.h"
void BuildCondition::read(ByteBuffer* buffer)
{
		this->c_id=buffer->readUTF();
		this->build_name=buffer->readUTF();
		this->build_type=buffer->readShort();
		this->build_num=buffer->readShort();
		this->condition_level=buffer->readShort();
}
